set.mm

```diff
@@ -435042,7 +435042,7 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
   latexdef "ZZring" as "\mathrm{ZZring}";
 htmldef "RR^" as "&#8477;^";
   althtmldef "RR^" as "&#8477;^";
-  latexdef "RR^" as "{\mathrm{RR^}}";
+  latexdef "RR^" as "\mathrm{RR^}";
```

Comment:
It should probably be "\mathbb{R}\hat{\;}" (ping @tirix)

Reply:
Sorry, I meant "\mathbb{R}\hat{\;}"

Reply:
ok, that's not me: there should be a backslash before the semicolon

Reply:
Yes! Good catch.